com.github.Rirusha.Cassette.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "com.github.Rirusha.Cassette",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.vala"
+    ],
+    "command" : "cassette",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--filesystem=~/.local/share",
```

Comment:
why?

Reply:
"--socket=wayland" and "--device=dri" because of a GNOME Template. "--socket=pulseaudio" because of an audioplayer.

Reply:
The question is for the last line, `--filesystem=~/.local/share`

Reply:
Deleted in last commit, thanks